api/Documents/methods.js

```diff
@@ -2,9 +2,9 @@
 
 import { Meteor } from 'meteor/meteor';
 import { check, Match } from 'meteor/check';
-import Documents from './Documents';
 import handleMethodException from '../../modules/handleMethodException';
 import rateLimit from '../../modules/rateLimit';
+import Documents from './';
```

Comment:
Let's keep the collections labeled by their names (e.g., `Documents.js`) instead of as index.js files. Reading this line back (from the perspective of a beginner) makes me instantly confused.

Reply:
Yeah I wasn't sure about this one. Even for someone who knows what this is, it's still weird.

So I see a few options

1. Leaving it with comments to explain what is happening
2. Having a Documents file and an index file. Then api imports won't be confusing, and we can still import elsewhere without doing Documents/Documents
3. Only have a Documents file. This means we would have to import elsewhere by doing Documents/Documents

Reply:
Even though it's verbose, #3 seems like the best option for the sake of clarity.

Reply:
Alright. I'll put it back to 3 and then merge this in.